projectlicense.md

```diff
@@ -1,61 +1,51 @@
+   may be used to endorse or promote products derived from this software
+   without specific prior written permission.
 
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
-ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
 FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
 DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-## About the Jupyter Development Team
-
-The Jupyter Development Team is the set of all contributors to the Jupyter project.
-This includes all of the Jupyter Subprojects, which are the different repositories
-under the [jupyter](https://github.com/jupyter/) GitHub organization.
-
-The core team that coordinates development on GitHub can be found here:
-https://github.com/jupyter/.
-
-## Our copyright policy
+## Explanation of Project Jupyter copyright policy
 
-Jupyter uses a shared copyright model. Each contributor maintains copyright
-over their contributions to Jupyter. But, it is important to note that these
-contributions are typically only changes to the repositories. Thus, the Jupyter
-source code, in its entirety is not the copyright of any single person or
-institution.  Instead, it is the collective copyright of the entire Jupyter
-Development Team.  If individual contributors want to maintain a record of what
-changes/contributions they have specific copyright on, they should indicate
-their copyright in the commit message of the change, when they commit the
-change to one of the Jupyter repositories.
+Project Jupyter uses a shared copyright model. Each contributor maintains
+copyright over their contributions to Project Jupyter. These contributions are
+typically only changes to the repositories. Thus, Project Jupyter source code,
+in its entirety, is not the copyright of any single person or institution.
+Instead, Project Jupyter source code is the collective copyright of its
+contributors. If individual contributors want to maintain a record of what
+changes or contributions they have specific copyright on, they should indicate
+their copyright in the commit message of the change when they commit the
```

Comment:
I don't have an objection to this language because it specifically says this is to maintain a record. It still remains the case that a copyright holders rights are intact irrespective of whether they explicitly say so.

The language in this clause does make it sound a little bit like those copyright notices *should* be in commit messages, but I think a close reading of this is actually correct, so I am not proposing a change. I am commenting here for clarity and also to invite disagreement if I've misunderstood.

Reply:
I had the same thoughts - the existing language was technically fine (though a bit awkward grammatically?), so I hesitated to change it too much.